e2e/utils/strategy/template.test.ts

```diff
@@ -2,9 +2,9 @@ import { describe, test, expect } from 'vitest';
 import { createDebugStrategy } from './template';
 
 describe('Test create strategy template', () => {
-  // Limit
-  test('limitBuy', () => {
-    const strategy = createDebugStrategy.limitBuy('ETH->DAI', {
+  // Recurring Limit
+  test('recurringLimitBuy', () => {
```

Comment:
I think you shouldn't change these tests. The mapping is like that :
- limitBuy -> disposable limit buy
- limitBuySell -> recurring limit_limit

The goal of these function was to build easily a testCase by doing : 
```
[
  limitBuy('ETH->DAI', { price: '100', budget: '100' }),
  ...
]
```
But it might have been premature work, so let's keep it as it was before.